src/components/MAPI/organizations/AccessControl/AccessControlRoleSubjects.tsx

```diff
@@ -145,6 +145,11 @@ const mapValueToSetItem = (stateValue: IStateValue) => (
   };
 };
 
+const sortByName = (
```

Comment:
Nitpicky: This doesn't really sort anything, right? It just compares 2 subjects

Reply:
Good point! I'll rename it to `compareSubjects` so it's more appropriate